app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="sleep_time">Sleep time in minutes</string>
```

Comment:
The dialog already has the word "minutes" after the text box. I don't think this is needed.

Reply:
> The dialog already has the word "minutes" after the text box. I don't think this is needed.

Removed 👍

Reply:
This string is still new, no need to add it when changing the dialog layout.